telegram/chat.py

```diff
@@ -277,6 +277,19 @@ def set_administrator_custom_title(self, *args: Any, **kwargs: Any) -> bool:
         """
         return self.bot.set_chat_administrator_custom_title(self.id, *args, **kwargs)
 
+    def unpin_all_chat_messages(self, *args: Any, **kwargs: Any) -> bool:
```

Comment:
can you add this to `User`, too, as we now can pin in private chats?

Reply:
Oh right, will do

Reply:
Should I also add `pin()`, `unpin()` and `unpin_all()` shortcuts in `CallbackQuery`?

Reply:
`pin` and `unpin` make sense, `unpin_all` not reall imo.

Reply:
uh, please call them `(un)_pin_message`, as we're not pinning the callback query ;)